examples/object_detection/android-jetpack-compose/app/src/main/java/com/example/objectdetection/objectdetector/ObjectDetectorHelper.kt

```diff
@@ -0,0 +1,377 @@
+/*
+ * Copyright 2022 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Update this licensing to 2023, then add it to every Kotlin file. You can see the current example for examples of where to put the licensing headers - xml and gradle files have them, too.

Reply:
Done